installer/Migration/V4_8_0/Migration.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace OrangeHRM\Installer\Migration\V4_8_0;
+
+use OrangeHRM\Installer\Util\V1\AbstractMigration;
+
+class Migration extends AbstractMigration
+{
+    /**
+     * @inheritDoc
+     */
+    public function up(): void
+    {
+        $this->insertConfig('help.url', 'https://opensourcehelp.orangehrm.com'); //has access issues.
+        $this->insertConfig('help.processorClass', 'ZendeskHelpProcessor');
+
+        $this->updateConfig('4.8', 'instance.version');
```

Comment:
remove this also, will define version in `getVersion`

Reply:
Removed